contrib/epee/src/readline_buffer.cpp

```diff
@@ -167,30 +165,42 @@ static int handle_enter(int x, int y)
   char* line = NULL;
```

Comment:
This wont work though as `rl_copy_text` malloc's the data and the `unique_ptr` default deleter uses `delete x` not `free(x)`. Thus a custom deleter would be needed.